specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/billing.json

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
/resource-manager/Microsoft.Billing/**_privatepreview_**/2020-11-01-privatepreview/billing.json 
Is this folder structure correct? Never seen any RP introducing privatepreview folder. Thay all keep this under preview.